### PR TITLE
Call mkinitrd

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

**Comment by teclator** [comment]
LGTM

File: test/bootloader_finish_client_test.rb [hunk 1]
```diff
@@ -2,166 +2,121 @@
...
+      subject.write
+    end
+
+    it "writes system configuration" do
+      expect(@system_bl).to receive(:write)
+
+      subject.write
+    end
+
+    it "return true if everything goes as expect" do
```
Comment (teclator) [inline test/bootloader_finish_client_test.rb]: returns

File: test/bootloader_finish_client_test.rb [hunk 2]
```diff
@@ -2,166 +2,121 @@
...
+    end
+
+    context "in Mode update" do
+      before do
+        Yast.import "Mode"
+
+        allow(Yast::Mode).to receive(:update).and_return(true)
+      end
+
+      it "do nothing if bootloader config is not read or proposed, so no changes done" do
```
Comment (teclator) [inline test/bootloader_finish_client_test.rb]: does

File: test/bootloader_finish_client_test.rb [hunk 3]
```diff
@@ -2,166 +2,121 @@
...
+      end
+    end
+
+    context "when kexec is requested" do
+      before do
+        allow(Yast::Linuxrc).to receive(:InstallInf).with("kexec_reboot")
+          .and_return("1")
+      end
+
+      it "prepare kexec environment" do
```
Comment (teclator) [inline test/bootloader_finish_client_test.rb]: prepares

**Comment by jreidinger** [comment]
thanks, grammar fixed

**Pull request merged**